vendor/zkllvm-metacraft-circuits/src/circuit_utils/static_vector.h

```diff
@@ -0,0 +1,187 @@
+#pragma once
+
+#include <array>
+#include <cstdint>
+#include <initializer_list>
+
+using Byte = unsigned char;
+
+template<typename T, std::size_t CAPACITY = 128, bool INITIALLY_FULL = false>
```

Comment:
same here

Reply:
I searched all files and I found and fixed one inconsistency in byte_utils.h as well. All template arguments should be consistent now, thanks @Owliie !